### ISSUE TITLE
addKtlintFormatGitPreCommitHook does not run in all classes of a multi-module application

### ISSUE THREAD

**Issue by gustavoterras** [JLLeitschuh/ktlint-gradle#449]
Using `addKtlintFormatGitPreCommitHook` the commnad `./gradlew ktlintFormat` is not executed in all classes of a multi-module application.

**project build.gradle.kts**
`// Top-level build file where you can add configuration options common to all sub-projects/modules.
buildscript {
    repositories {
        google()
        jcenter()
        maven("https://plugins.gradle.org/m2/")
    }
    dependencies {
        classpath("com.android.tools.build:gradle:4.1.2")
        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.30")
        classpath("org.jlleitschuh.gradle:ktlint-gradle:10.0.0")
        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

allprojects {
    repositories {
        google()
        jcenter()
    }

    apply(plugin = "org.jlleitschuh.gradle.ktlint")
}

tasks {
    val clean by registering(Delete::class) {
        delete(buildDir)
    }
}
`

<img width="725" alt="Captura de Tela 2021-03-05 às 16 32 14" src="https://user-images.githubusercontent.com/8082012/110164795-a16ee100-7dd0-11eb-96a0-1fab2324a5f1.png">

The only formatted file is the one flagged in the print, but there are more files modified in the commit.

**Comment by Tapchicoma** [comment]
Do other files contains code violations as well?

Could you create a project that reproduces this issue?

**Comment by gustavoterras** [comment]
All commit files have violations

Check my sample project [ktlint-Git-Hooked](https://github.com/gustavoterras/ktlint-Git-Hooked)

**Comment by gustavoterras** [comment]
@Tapchicoma is there a problem with my implementation?

**Comment by Tapchicoma** [comment]
Yes, I see that running git hook in this case does not modify all files with violations. Running `./gradlew ktlintFormat` on the other hand fixes all violations.

Probably the issues is in that your project has Gradle root project in the subdirectory and proper filtering is not applied on running format task. Will investigate it further.
